generator/standalone.h

```diff
@@ -0,0 +1,21 @@
+#ifndef GAPIC_GENERATOR_CPP_GENERATOR_STANDALONE_H_
+#define GAPIC_GENERATOR_CPP_GENERATOR_STANDALONE_H_
+
+#include <google/protobuf/compiler/plugin.h>
+#include <google/protobuf/compiler/plugin.pb.h>
+
+namespace google {
+namespace api {
+namespace codegen {
+
+int StandaloneMain(int argc, const char* const argv[],
+                   google::protobuf::compiler::CodeGenerator* generator);
```

Comment:
Why pass by pointer instead of by reference?

Reply:
To make this `StandaloneMain` function accept same arguments as `PluginMain`  (i.e. for consistency).